src/io/flutter/actions/AttachDebuggerAction.java

```diff
@@ -104,14 +104,15 @@ public void update(AnActionEvent e) {
       return;
     }
     boolean enabled;
-    RunnerAndConfigurationSettings settings = RunFlutterAction.getRunConfigSettings(e);
+    RunnerAndConfigurationSettings settings = RunManagerEx.getInstanceEx(project).getSelectedConfiguration();
     if (settings == null) {
       enabled = false;
     }
     else {
       RunConfiguration configuration = settings.getConfiguration();
       enabled = configuration instanceof SdkRunConfig;
     }
+    e.getPresentation().setVisible(true);
```

Comment:
I assume that we do want this visible even if enabled == false? We'll want it visible for Flutter enabled projects, just not enabled if if doesn't apply to the currently running app?

Reply:
Yes, it would be disabled if the run config is for Android, and enabled when a Flutter run config is selected.